corehq/apps/geospatial/views.py

```diff
@@ -441,10 +441,17 @@ def get_users_with_gps(request, domain):
 @method_decorator(toggles.GEOSPATIAL.required_decorator(), name="dispatch")
 class CasesReassignmentView(BaseDomainView):
     urlname = "reassign_cases"
+    MAX_REASSIGNMENT_REQUEST_CASES = 100
```

Comment:
This is based on the pagination on the UI, right?

Reply:
That is correct